rfcs/0002-ethereum-tracing-cache.md

```diff
@@ -0,0 +1,61 @@
+# RFC-0002: Ethereum Tracing Cache
+
+<dl>
+  <dt>Author</dt>
+  <dd>Zac Burns</dd>
+
+  <dt>RFC pull request</dt>
+  <dd><a href="https://github.com/graphprotocol/rfcs/pull/4">https://github.com/graphprotocol/rfcs/pull/4</a></dd>
+
+  <dt>Obsoletes (if applicable)</dt>
+  <dd>None</dd>
+
+  <dt>Date of submission</dt>
+  <dd>2019-12-13</dd>
+
+  <dt>Date of approval</dt>
+  <dd>YYYY-MM-DD</dd>
+
+  <dt>Approved by</dt>
+  <dd>First Person, Second Person</dd>
+</dl>
+
+## Summary
+
+This RFC proposes the creation of a local ethereum tracing cache to speed up indexing of subgraphs which use block and/or call handlers.
```

Comment:
Here's comes the nit-picky Jannis (you'll get used to me): Can we spell ethereum `Ethereum` with a capital E? Mainly because it's its own name, like Greenpeace or GitHub.